src/cmd/run.js

```diff
@@ -333,6 +333,6 @@ export class ExtensionRunner {
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
     const {firefoxApp, firefox} = this;
-    return firefoxApp.run(profile, {firefox});
+    return firefoxApp.run(profile, {firefoxBinary: firefox || null});
```

Comment:
does this need to pass `null` to satisfy the flow type or something? Could it just be `{firefoxBinary: firefox}` ?